src/dstack/_internal/server/schemas/runs.py

```diff
@@ -26,6 +26,7 @@ class GetRunRequest(CoreModel):
 
 class GetRunPlanRequest(CoreModel):
     run_spec: RunSpec
+    max_offers: Optional[int] = Field(description="The maximum number of offers to return")
```

Comment:
Should have limits

Reply:
@r4victor Do you think 10000 upper limit is ok?

Reply:
probably